Some weights of the model checkpoint at bert-base-cased were not used when initializing BertModel: ['cls.predictions.bias', 'cls.predictions.decoder.weight', 'cls.seq_relationship.weight', 'cls.seq_relationship.bias', 'cls.predictions.transform.LayerNorm.bias', 'cls.predictions.transform.LayerNorm.weight', 'cls.predictions.transform.dense.weight', 'cls.predictions.transform.dense.bias']
- This IS expected if you are initializing BertModel from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing BertModel from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
Some weights of GPT2LMHeadModel were not initialized from the model checkpoint at gpt2 and are newly initialized: ['h.11.ln_cross_attn.weight', 'h.5.crossattention.c_proj.weight', 'h.2.crossattention.q_attn.weight', 'h.6.crossattention.c_proj.bias', 'h.2.crossattention.c_proj.bias', 'h.10.ln_cross_attn.weight', 'h.8.crossattention.c_attn.weight', 'h.3.crossattention.bias', 'h.5.crossattention.masked_bias', 'h.2.crossattention.c_attn.weight', 'h.0.crossattention.c_attn.weight', 'h.10.crossattention.c_attn.weight', 'h.4.crossattention.c_attn.weight', 'h.9.crossattention.c_proj.weight', 'h.3.crossattention.masked_bias', 'h.8.crossattention.masked_bias', 'h.8.crossattention.c_proj.bias', 'h.7.crossattention.q_attn.weight', 'h.2.crossattention.c_proj.weight', 'h.2.crossattention.masked_bias', 'h.2.ln_cross_attn.weight', 'h.0.crossattention.masked_bias', 'h.9.crossattention.bias', 'h.7.ln_cross_attn.weight', 'h.8.ln_cross_attn.weight', 'h.10.crossattention.masked_bias', 'h.10.crossattention.c_proj.bias', 'h.0.crossattention.c_proj.bias', 'h.0.ln_cross_attn.weight', 'h.6.ln_cross_attn.weight', 'h.4.crossattention.c_proj.weight', 'h.6.crossattention.bias', 'h.9.crossattention.c_attn.weight', 'h.11.crossattention.bias', 'h.4.crossattention.q_attn.weight', 'h.6.crossattention.masked_bias', 'h.1.crossattention.bias', 'h.9.crossattention.masked_bias', 'h.3.crossattention.c_proj.weight', 'h.1.crossattention.c_proj.bias', 'h.3.crossattention.c_proj.bias', 'h.8.crossattention.c_proj.weight', 'h.7.crossattention.bias', 'h.9.crossattention.c_proj.bias', 'h.7.crossattention.c_proj.weight', 'h.4.crossattention.c_proj.bias', 'h.5.ln_cross_attn.weight', 'h.0.crossattention.q_attn.weight', 'h.1.crossattention.q_attn.weight', 'h.6.crossattention.c_attn.weight', 'h.6.crossattention.c_proj.weight', 'h.1.crossattention.c_proj.weight', 'h.10.crossattention.c_proj.weight', 'h.0.crossattention.bias', 'h.5.crossattention.q_attn.weight', 'h.11.crossattention.c_proj.bias', 'h.8.crossattention.q_attn.weight', 'h.7.crossattention.c_attn.weight', 'h.8.crossattention.bias', 'h.10.crossattention.q_attn.weight', 'h.5.crossattention.c_attn.weight', 'h.11.crossattention.q_attn.weight', 'h.1.ln_cross_attn.weight', 'h.4.ln_cross_attn.weight', 'h.9.crossattention.q_attn.weight', 'h.9.ln_cross_attn.weight', 'h.1.crossattention.masked_bias', 'h.4.crossattention.masked_bias', 'h.3.ln_cross_attn.weight', 'h.3.crossattention.c_attn.weight', 'h.7.crossattention.masked_bias', 'h.11.crossattention.masked_bias', 'h.1.crossattention.c_attn.weight', 'h.4.crossattention.bias', 'h.7.crossattention.c_proj.bias', 'h.6.crossattention.q_attn.weight', 'h.10.crossattention.bias', 'h.5.crossattention.bias', 'h.11.crossattention.c_attn.weight', 'h.11.crossattention.c_proj.weight', 'h.0.crossattention.c_proj.weight', 'h.3.crossattention.q_attn.weight', 'h.2.crossattention.bias', 'h.5.crossattention.c_proj.bias']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
loaded 232 training samples
loaded 53 validation samples
loaded 6 test samples
created model
Begin training!
Traceback (most recent call last):
  File "/home/mitryand/.local/lib/python3.10/site-packages/torch/utils/data/_utils/collate.py", line 178, in default_collate
    return elem_type([default_collate(samples) for samples in transposed])
  File "/home/mitryand/.local/lib/python3.10/site-packages/torch/utils/data/_utils/collate.py", line 178, in <listcomp>
    return elem_type([default_collate(samples) for samples in transposed])
  File "/home/mitryand/.local/lib/python3.10/site-packages/torch/utils/data/_utils/collate.py", line 141, in default_collate
    return torch.stack(batch, 0, out=out)
TypeError: expected Tensor as element 6 in argument 0, but got int

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mitryand/.local/lib/python3.10/site-packages/torch/utils/data/_utils/collate.py", line 160, in default_collate
    return elem_type({key: default_collate([d[key] for d in batch]) for key in elem})
  File "/home/mitryand/.local/lib/python3.10/site-packages/torch/utils/data/_utils/collate.py", line 160, in <dictcomp>
    return elem_type({key: default_collate([d[key] for d in batch]) for key in elem})
  File "/home/mitryand/.local/lib/python3.10/site-packages/torch/utils/data/_utils/collate.py", line 181, in default_collate
    return [default_collate(samples) for samples in transposed]
  File "/home/mitryand/.local/lib/python3.10/site-packages/torch/utils/data/_utils/collate.py", line 181, in <listcomp>
    return [default_collate(samples) for samples in transposed]
  File "/home/mitryand/.local/lib/python3.10/site-packages/torch/utils/data/_utils/collate.py", line 141, in default_collate
    return torch.stack(batch, 0, out=out)
TypeError: expected Tensor as element 6 in argument 0, but got int

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mitryand/.local/lib/python3.10/site-packages/torch/utils/data/_utils/collate.py", line 178, in default_collate
    return elem_type([default_collate(samples) for samples in transposed])
  File "/home/mitryand/.local/lib/python3.10/site-packages/torch/utils/data/_utils/collate.py", line 178, in <listcomp>
    return elem_type([default_collate(samples) for samples in transposed])
  File "/home/mitryand/.local/lib/python3.10/site-packages/torch/utils/data/_utils/collate.py", line 141, in default_collate
    return torch.stack(batch, 0, out=out)
TypeError: expected Tensor as element 6 in argument 0, but got int

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mitryand/EECS-595-Final-Project-Style-Transfer/train_model.py", line 216, in <module>
    main(params)
  File "/home/mitryand/EECS-595-Final-Project-Style-Transfer/train_model.py", line 197, in main
    model = train(model, train_dataloader, eval_dataloader, params, input_tokenizer, output_tokenizer)
  File "/home/mitryand/EECS-595-Final-Project-Style-Transfer/train_model.py", line 112, in train
    for batch in train_dataloader:
  File "/home/mitryand/.local/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 681, in __next__
    data = self._next_data()
  File "/home/mitryand/.local/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 721, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
  File "/home/mitryand/.local/lib/python3.10/site-packages/torch/utils/data/_utils/fetch.py", line 52, in fetch
    return self.collate_fn(data)
  File "/home/mitryand/.local/lib/python3.10/site-packages/torch/utils/data/_utils/collate.py", line 163, in default_collate
    return {key: default_collate([d[key] for d in batch]) for key in elem}
  File "/home/mitryand/.local/lib/python3.10/site-packages/torch/utils/data/_utils/collate.py", line 163, in <dictcomp>
    return {key: default_collate([d[key] for d in batch]) for key in elem}
  File "/home/mitryand/.local/lib/python3.10/site-packages/torch/utils/data/_utils/collate.py", line 181, in default_collate
    return [default_collate(samples) for samples in transposed]
  File "/home/mitryand/.local/lib/python3.10/site-packages/torch/utils/data/_utils/collate.py", line 181, in <listcomp>
    return [default_collate(samples) for samples in transposed]
  File "/home/mitryand/.local/lib/python3.10/site-packages/torch/utils/data/_utils/collate.py", line 141, in default_collate
    return torch.stack(batch, 0, out=out)
TypeError: expected Tensor as element 6 in argument 0, but got int
